ionic2/app/pages/poke-detail/poke-detail.page.html

```diff
@@ -21,11 +25,8 @@
             <span *ngIf="pokemon.gender.abbreviation=='f'">, female</span>
             <span *ngIf="pokemon.gender.abbreviation=='m'">, male</span>
           </div>
-          <div id="types"><poke-type [type]="pokemon.types"></poke-type></div>
-          <!--div id="rarity">
-            <poke-rarity-badge [rarity]="pokemon.appearanceLikelihood"></poke-rarity-badge>
-            <span style="opacity: 0.5">(Rarity: Soon badge instead of stars)</span>
-          </div-->
+          <div id="types"><poke-type [types]="pokemon.types"></poke-type></div>
```

Comment:
Still bad. The component should be called `<poke-type>` it should have a `type` property and it should accept a single value (not an array)